ur5_moveit_config/launch/moveit_rviz.launch

```diff
@@ -11,6 +12,7 @@
   <node name="$(anon rviz)" launch-prefix="$(arg launch_prefix)" pkg="rviz" type="rviz" respawn="false"
 	args="$(arg command_args)" output="screen">
     <rosparam command="load" file="$(find ur5_moveit_config)/config/kinematics.yaml"/>
+    <param name="arm_prefix" value="$(arg arm_prefix)"/>
```

Comment:
I asked about this on `moveit-users` some time ago, see [parameters for ikfast plugins using private NodeHandles](https://groups.google.com/d/msg/moveit-users/jlneWn9uumo/aceglJNfkUwJ). The response by @sachinchitta is [here](https://groups.google.com/d/msg/moveit-users/gk1lPiP8ylo/85Ba9BVKptkJ).